src/main/java/org/jabref/logic/layout/format/HTMLChars.java

```diff
@@ -23,6 +23,15 @@ public class HTMLChars implements LayoutFormatter {
      * */
     private static final Pattern HTML_ENTITY_PATTERN = Pattern.compile("&(?!(?:[a-z0-9]+|#[0-9]{1,6}|#x[0-9a-fA-F]{1,6});)");
 
+    private boolean keepCurlyBraces = false;
+
+    @Override
+    public void setArgument(String arg) {
+        if ("keepCurlyBraces".equals(arg)) {
```

Comment:
Please use equalsIgnoreCase. The CPU usage is neglectable in comparison to user experience.